dev-core

```diff
@@ -4,8 +4,8 @@ echo "Building mesg/core:local..."
 docker build -t mesg/core:local --build-arg version="local" .
 
 # chain these to stop execution on failure
-./dev-cli stop && \
-./dev-cli start && \
+./dev-cli stop
+./dev-cli start
```

Comment:
Why removing the chain of execution? as the comment says, it's to stop on failure.

Reply:
@ilgooz Please revert the change

Reply:
it was due to a bug while stopping core. reverted it back. i'll open an issue about the bug later